src/fuzz.rs

```diff
@@ -265,8 +278,9 @@ mod tests {
         s9a: &'static str,
         s10: &'static str,
         s10a: &'static str,
-        // TODO: Test silly corner cases,
+        #[allow(dead_code)]
         cirque_strings: &'static [&'static str; 6],
```

Comment:
I added these blindly while porting the unit tests, however neither the original nor our ported tests make use of these. Shall we delete them?

Reply:
I just haven't done fuzz yet, but probably.